src/status_im/ui/components/label.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.ui.components.label
+  (:require [status-im.ui.components.react :as components]
+            [status-im.i18n :as i18n]
+            [status-im.ui.screens.home.styles :as home.styles]
+            [status-im.utils.core :as utils]))
+
+(def default-styles
+  {:letter-spacing -0.2
+   :font-size      14})
+
+(defn label
+  ([i18n-key] (label {} i18n-key))
+  ([styles i18n-key] (components/text {:style (merge default-styles styles)}
```

Comment:
i think it should return vector `[`components/text {:style (merge default-styles styles)}